### PR TITLE
4156 Improved error message for coverage metadata

### PR DESCRIPTION
**Pull request by pkdash**
### Pull Request Checklist: 
- [x] Positive Test Case Written by Dev

- [x] Automated Testing

- [x] Sufficient User and Developer Documentation

- [x] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. Create a resource.
2. In the resource editing page:

   2.1. Try to update the resource level coverage metadata. Enter a name for the Place/Area name field only. Then click the 'Save changes' button. Coverage metadata update should fail and you should see data missing for specific coverage extent fields. Try to enter a suitable value for either the Longitude or the Latitude field. Then again try to save the changes. Update should fail and you should see data missing for specific coverage extent field.

   2.2. Try to update the temporal coverage for the resource. Enter data for the Start date field only. Then click the  'Save changes'  button. Metadata update should fail and you see a message that data is missing for the end date. 

   2.3. Upload a text file. Make that a single file aggregation. Then try to update the aggregation level coverage metadata as per step 2.1 and 2.2 above.

### CODE REVIEW

**Comment by hydrocheck** [comment]
| METRIC | VALUE |
|:-----:|:-----:|
| NEW SECURITY HOTSPOTS | 0 |
| NEW CODE SMELLS | 1 |
| NEW BUGS | 0 |
| NEW VULNERABILITIES | 0 |
| NEW DUPLICATED LINES DENSITY | 0.0 |

[https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4433](https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4433)

**Comment by hydrocheck** [comment]
| METRIC | VALUE |
|:-----:|:-----:|
| NEW VULNERABILITIES | 0 |
| NEW CODE SMELLS | 0 |
| NEW BUGS | 0 |
| NEW SECURITY HOTSPOTS | 0 |

[https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4433](https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4433)

File: hs_core/forms.py [hunk 1]
```diff
@@ -1045,13 +1045,14 @@ def clean(self):
             if 'elevation' in temp_cleaned_data:
                 del temp_cleaned_data['elevation']
 
-            for limit in ('northlimit', 'eastlimit', 'southlimit', 'westlimit'):
+            box_fields_map = {"northlimit": "north latitude", "southlimit": "south latitude",
+                              "eastlimit": "east longitude", "westlimit": "west longitude"}
+            for limit in box_fields_map.keys():
```
Comment (hyi) [inline hs_core/forms.py:1050]: @pkdash You can just use ```for limit in box_fields_map```

**Review by hyi — APPROVED** [review]
@pkdash Added one comment just as a FYI. I am approving the PR.

File: hs_core/forms.py (hunk 1 above)
Comment (pkdash) [reply hs_core/forms.py:1050]: @hyi True.

**Pull request merged**